content/docs/tutorials/basic/ruby.md

```diff
@@ -279,12 +268,12 @@ so that clients can actually use our service. The following snippet shows how we
 do this for our `RouteGuide` service:
 
 ```ruby
-addr = "0.0.0.0:8080"
+port = '0.0.0.0:50051'
 s = GRPC::RpcServer.new
-s.add_http2_port(addr, :this_port_is_insecure)
-logger.info("... running insecurely on #{addr}")
+s.add_http2_port(port, :this_port_is_insecure)
+GRPC.logger.info("... running insecurely on #{port}")
 s.handle(ServerImpl.new(feature_db))
-s.run_till_terminated
+s.run_till_terminated_or_interrupted([1, 'int', 'SIGQUIT'])
```

Comment:
can we also copy over the comment from `route_guide_server.rb`:

```
  # Runs the server with SIGHUP, SIGINT and SIGQUIT signal handlers to 
  #   gracefully shutdown.
  # User could also choose to run server via call to run_till_terminated
```

Reply:
Thanks! Have updated.